src/settings/settings.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  color: $white-color;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.section {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  height: 100%;
+  width: 50%;
+  background-color: $black-background-color;
```

Comment:
Just `black-color` would be enough. Color is a color - you can use it for many things, not only for background 😄